lib/tasks/sidekiq.rake

```diff
@@ -43,14 +43,29 @@ namespace :sidekiq do
         dest: RedisJobSet.new(redis: dest_redis),
         logger: logger,
         progress: progress,
-        )
+      )
 
       delayed_queue = SidekiqMover.new(
         source: SidekiqDelayedQueue.new(redis: source_redis),
-        dest:   SidekiqDelayedQueue.new(redis: dest_redis),
+        dest: SidekiqDelayedQueue.new(redis: dest_redis),
         logger: logger,
         progress: progress
-        )
+      )
+      # https://github.com/mperham/sidekiq/blob/63ee43353bd3b753beb0233f64865e658abeb1c3/lib/sidekiq/api.rb#L641
+      retry_queue = SidekiqMover.new(
+        source: SidekiqDelayedQueue.new(key: 'retry', redis: source_redis),
+        dest: SidekiqDelayedQueue.new(key: 'retry', redis: dest_redis),
+        logger: logger,
+        progress: progress,
+      )
+
+      # https://github.com/mperham/sidekiq/blob/63ee43353bd3b753beb0233f64865e658abeb1c3/lib/sidekiq/api.rb#L656
```

Comment:
https://github.com/mperham/sidekiq/wiki/API#dead might be a better link?

Reply:
I don't know if you saw my comment on the retry thing before it became 'outdated' but basically, I feel like what matters here is the _implementation_. We want to know that it's `retry` and not `retries`.

Reply:
Perhaps this is naive, but wouldn’t it be safer to retrieve the queue name with `Sidekiq::RetrySet.new.name`? Then you’re guaranteed to have the right name.